locksmith/__tests__/controllers/userController.test.ts

```diff
@@ -101,7 +103,29 @@ describe('User Controller', () => {
     describe('when the provided email does not within the existing persistence layer', () => {
       it('returns an error code', async () => {
         let emailAddress = 'non-existing@example.com'
-        let response = await request(app).get(`/users/${emailAddress}/privatekey`)
+        let response = await request(app).get(
+          `/users/${emailAddress}/privatekey`
+        )
+        expect(response.statusCode).toBe(400)
+      })
+    })
+  })
+
+  describe("retrieving a user's recovery phrase", () => {
+    describe('when the user exists', () => {
+      it("returns the user's recovery phrase", async () => {
+        let response = await request(app).get(
+          '/users/user@example.com/recovery_phrase'
+        )
+        expect(response.body).toEqual({ recoveryPhrase: 'a recovery phrase' })
+      })
+    })
+
+    describe('when the user does not exist', () => {
```

Comment:
I think similarly to how we server a "fake" encrypted private key we should serve a fake recovery key here if the user does not exist.

Reply:
Agreed, I've noted this as a next step in the description of this pr!

Reply:
oups! sorry about missing it...